osquery/tables/system/linux/tests/pcidb_tests.cpp

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "osquery/tables/system/linux/pci_devices.h"
+
+using namespace testing;
```

Comment:
Please avoid `using namespace` constructions if it possible, because it make code unreadable quite often. Because of very non obvious C++ name resolving policy.